Tests/webcomponents/js/ez-universal-discovery.js

```diff
@@ -0,0 +1,409 @@
+
+        it('should have a `minDiscoverDepth` property', function () {
+            assert.equal(
+                universalDiscovery.getAttribute('min-discover-depth'),
+                universalDiscovery.minDiscoverDepth
+            );
+        });
+
+        it('should have a `visibleMethod` property', function () {
+            assert.equal(
+                universalDiscovery.getAttribute('visible-method'),
+                universalDiscovery.visibleMethod
+            );
+        });
+        describe('`multiple` property', function () {
+            it('should have a `multiple` property', function () {
+                assert.equal(
+                    universalDiscoveryMultiple.hasAttribute('multiple'),
+                    universalDiscoveryMultiple.multiple
+                );
+            });
+
+            it('`multiple` property should be false by default', function () {
+                assert.isFalse(
+                    universalDiscovery.multiple
+                );
+            });
+        });
+    });
+
+    describe('YUI stack loading', function () {
+        let renderViewSpy;
+
+        function assertViewRendered(spy, Y) {
+            assert.isOk(
+                spy.callCount
+            );
+            assert.equal(
+                spy.firstCall.args[0],
+                Y.eZ.UniversalDiscoveryView,
+                'The UniversalDiscoveryView should have been rendered'
+            );
+        }
+
+        afterEach(function () {
+            delete window.eZ;
+            renderViewSpy.restore();
+        });
+
+        describe('`ez-yui-app:ready` event', function () {
+            beforeEach(function () {
+                universalDiscovery = fixture('BasicTestFixture');
+                defineGlobals();
+                renderViewSpy = sinon.spy(window.eZ.YUI.app, 'renderSideView');
+            });
+
+            it('should render the universal discovery', function () {
+                document.dispatchEvent(new CustomEvent('ez:yui-app:ready'));
+
+                assertViewRendered(renderViewSpy, window.eZ.YUI.Y);
+            });
+        });
+
+        describe('already loaded', function () {
+            beforeEach(function () {
+                defineGlobals();
+                renderViewSpy = sinon.spy(window.eZ.YUI.app, 'renderSideView');
+                universalDiscovery = fixture('BasicTestFixture');
+            });
+
+            it('should render the universal discovery right away', function () {
+                assertViewRendered(renderViewSpy, window.eZ.YUI.Y);
+            });
+        });
+    });
+
+    describe('YUI Universal Discovery Widget', function () {
+        let renderViewStub;
+        const container = document.createElement('div');
+        const location = getModelMock();
+        const content = getModelMock();
+        const contentType = getModelMock();
+
+        function assertEventParam(e) {
+            assert.equal(e.detail.selection.content, modelObject);
```

Comment:
strictEqual
in addition, it's a bit weird to reference a global variable here and the same each and every selection property